windows-installer/Package.en-us.wxl

```diff
@@ -8,9 +8,9 @@ This file contains the declaration of all the localizable strings.
   <String Id="InstallPathDlgTitle" Value="{\WixUI_Font_Title}Select Project Folder" />
   <String Id="InstallPathDlgDescription" Value="Click Next to install to the default folder or customize with options below." />
   <String Id="InstallDirDlgBannerBitmap" Value="WixUI_Bmp_Banner"/>
-  <String Id="SelectRootDrive" Value="Please choose the drive where you want to install IDEasy:"/>
-  <String Id="SelectRootDriveText" Value="Please choose the drive where you want to install IDEasy:"/>
+  <String Id="SelectCustomDrive" Value="Please choose the drive where you want to install IDEasy:"/>
+  <String Id="SelectCustomDriveText" Value="Please choose the drive where you want to install IDEasy:"/>
   <String Id="SelectInstallLocationText" Value="Select your preferred installation location:"/>
-  <String Id="InstallRootOption" Value="Install in Root Directory (e.g., C:\projects)"/>
-  <String Id="InstallHomeOption" Value="Install in Home Directory (e.g., C:\Users\YourUsername\projects)"/>
+  <String Id="InstallUserHomeOption" Value="Install in Home Directory (e.g., C:\Users\YourUsername\projects)"/>
+  <String Id="InstallCustomOption" Value="Install in Custom Directory (e.g., C:\projects)"/>
```

Comment:
Would be nice if this could be dynamic too.

Reply:
Thats gonna be tricky to do it, when it has already been rendered. But I changed it so that the `YourUsername` is replaced by the current user name and the drive is the `ROOTDRIVE` of the system. Thanks for the suggestion